src/quo2/components/messages/author/view.cljs

```diff
@@ -0,0 +1,65 @@
+(ns quo2.components.messages.author.view
+  (:require
+   [quo2.components.messages.author.style :as style]
+   [quo.react-native :as rn]
+   [quo2.components.markdown.text :as text]
+   [quo2.components.icon :as icons]
+   [status-im.utils.utils :as utils]))
+
+(def middle-dot "·")
+
+(defn author [{:keys [profile-name nickname chat-key ens-name time-str is-contact? is-verified? is-untrustworthy?]}]
```

Comment:
I think we can go without the `is-` prefix in Clojure because the question mark is enough.

Reply:
agree

Reply:
True, will update it